tests/visco_plastic_peierls_mei_viscosity_approx.prm

```diff
@@ -0,0 +1,125 @@
+# A test for the peierls creep viscositiy approximation  
+# with the visco_plastic material model. 
+# Following the general Peierls creep flow law of Mei 
+# et al. 2010 and assuming a stress of 900e6 Pa, the
+# exact viscosity is calculated in python through:
+#  import numpy as np
+#  A    = 1.4e-7/np.power(1e6,2) # Peierls creep prefactor 1/s 1/Pa^2 
+#  R    = 8.314   # gas constant [J/mol*K] 
+#  T    = 1073    # temperature [K]
+#  sig  = 1000e6  # stress [Pa] 
+#  sigp = 5.9e9   # Peierls creep stress [Pa]
+#  E    = 320e3   # Peierls creeep activation energy [J/mol]
+#  V    = 0       # Peierls creep activation volume
+#  P    = 0       # Pressure (Pa)
+#  n    = 2       # Peierls creep stress exponent
+#  p    = 0.5     # second fitting exponent
+#  q    = 1       # second fitting exponent
+# 
+#  Strain rate invariant:
+#    edot_ii = A*np.power(sig,n)*np.exp(-(E/(R*T))*np.power( (1-np.power((sig/sigp),p) ),q) )
+#            = 9.326722654162315e-11
+#
+#  Viscosity:
+#    visc = 0.5*sig/edot_ii = 5.22097951523685e+18
+#
+#  Fitting parameters for approximated viscosity:
+#    gamma = 0.15 # fitting parameter
+# 
+#  Exponential term in Peierls creep viscosity approximation
+#    s =  p*q*np.power(g,p)*np.power((1-np.power(g,p)),q-1)*(E/(R*T)) 
+#
+#  # Approximated viscosity
+#    visca = 0.5*gamma*sigp/np.power(A*np.power(gamma*sigp,n),1/(s+n))*np.power(edot_ii,(1/(s+n))-1)*np.exp((E/(R*T))*np.power((1 - np.power(gamma,p)),q)/(s+n)) 
+#          = 5.236440783251679e+18
```

Comment:
When I calculate this in my script and plugging in the same numbers for T, P, V, and edot, I get visc   = 5.360939941500902e+18 
which matches what the Aspect puts out, check values that you put into the python script.

Reply:
I get the same thing as well now. I forgot to calculate and update strain-rate for the approximation.